scripts/run_lighthouse_tests.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Lighthouse checks and store reports script."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import shutil
+import subprocess
+
+import python_utils
+from scripts import common
+from . import install_third_party_libs
+
+
+def setup_and_install_dependencies():
+    """Run the setup and installation scripts."""
```

Comment:
Runs* to be consistent with verb tense (descriptive style, see [here](http://google.github.io/styleguide/pyguide.html#383-functions-and-methods) for more information
). Generally for methods, you can imagine that there was a "This method" at the beginning of the sentence to get the descriptive style tense.

Reply:
Done!